index.html

```diff
@@ -583,24 +583,48 @@ <h3 class="ui header">Indices status and management</h3>
 
 
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+	<a name="reIndexes"></a>
+	<h2 class="ui dividing header">Reindex API</h2>
+	<h3 class="ui header">Simple Reindex Operation</h3>
+	<pre class="es1 es2"><code>POST /_reindex
```

Comment:
The ReIndex API was not available in Elasticsearch 1, so the es1 class should not be set here. In fact the whole "reindex" part should be enclosed in a div with "es2 es5" classes.